include/ibamr/IIMethod.h

```diff
@@ -168,6 +168,22 @@ class IIMethod : public IBStrategy
         void* ctx;
     };
 
+/*!
+     * Returns either the current configuration (at possibly different times in the timestep)
+     * or the reference configuration with a numeric vector
+     * 
+     */
+    libMesh::NumericVector<double>* 
+    getMeshCoordinatesNumeric(bool isCurrentConfiguration,std::string time,unsigned int part);
```

Comment:
```suggestion
    getMeshCoordinatesNumeric(bool current_configuration, std::string time, unsigned int part);
```
Also, let's not use strings to identify the time level. @abarret, do we have enums for this at this point?

Reply:
Strings are consistent with how time levels are identified in `IBFEMethod` and related classes. Enums for this are in #1688, but they are not widely implemented.

Reply:
Thinking some more about this --- I it might help if this function was named something like "interaction mesh coordinates".

BTW, ultimately the interaction mesh coordinates might not be either the reference or current coordinates, but some other coordinate system (e.g. the boundary of the solid mesh).

Reply:
And to that end, it might make sense to go ahead and introduce some new `LibMeshSystemVector`s to track the interaction configuration...